x-pack/test/security_functional/tests/security_check/unsecure_cluster_alert.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const find = getService('find');
+  const PageObjects = getPageObjects(['common']);
+  const esArchiver = getService('esArchiver');
+
+  describe('Unsecure Cluster Alert', function () {
+    before(async () => {
+      await esArchiver.emptyKibanaIndex();
+      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/logstash_functional');
+    });
+
+    after(async () => {
+      await esArchiver.unload('x-pack/test/functional/es_archives/logstash_functional');
+    });
+
+    it('will display when ES Security Plugin is disabled and there is at least one user created index with data', async () => {
+      await PageObjects.common.navigateToUrl('home');
+
+      const toastMessage: string = await (await find.byClassName('euiToast')).getVisibleText();
```

Comment:
__nit:__ we shouldn't rely on such generic class name in tests, instead we should pass `'data-test-subj': 'insecureClusterToast'` property to the `notifications.toasts.addWarning` and use `testSubjects` FTR service to retrieve required element.